src/stan_math_backend/Transform_Mir.ml

```diff
@@ -1,6 +1,92 @@
 open Core_kernel
 open Middle
 
+let opencl_triggers =
+  String.Map.of_alist_exn
+    [ ("normal_id_glm_lpdf",
+        ([0;1],
+          [ (* Array of conditions under which to move to OpenCL *)
+            ([1],[]) (* Argument 1 is data *)
+          ]
+        )
+      )
+    ; ("bernoulli_logit_glm_lpmf",
+        ([0;1], [([1],[])])
+      )
+    ; ("categorical_logit_glm_lpmf",
+        ([0;1], [([-1],[(1,UMatrix)])]) (* Always use OpenCL, when argument 1 is a matrix *)
```

Comment:
This one was removed since, did you comment to a previous commit?

I wanted to use -1 as kind of saying [-1] as a wildcard (no data needed), but then figure I dont need it.